README.md

```diff
@@ -5,7 +5,7 @@ yip is a command line tool used for doing in place modifications to a yaml objec
 Prerequisite: 
   * Install [go](https://golang.org/doc/install)
 
-Installation:
+Installation: (for Mac/Darwin)
   1. `git clone git@github.com:launchpadcentral/yip.git`
   2. `cd yip`  
   3. `mv main.go yip.go`
```

Comment:
I was going to comment about why we are changing this name, but I realized go install doesn't allow for setting output names. I wonder if building into .bin would be a better approach